modules/test/playwright/tests/cookies-banner-web/cookiesBannerScript.spec.ts

```diff
@@ -52,7 +54,6 @@ test('@LPD-25701 Cookie Banner Script', async ({
 		await htmlExample.waitFor({state: 'visible'});
 		await htmlExample.click();
 		await htmlExample.click();
-		await htmlExample.click();
```

Comment:
You said in https://github.com/liferay-frontend/liferay-portal/pull/4190#discussion_r1604099317 that triple click is the only way to edit. What changed?

Reply:
Yeah this section is flaky as well and I'm thinking of redoing this a different way in order to prevent all the flaky tasks.